diff --git a/node_modules/react-native-element-dropdown/lib/module/index.js b/node_modules/react-native-element-dropdown/lib/module/index.js
index f9175a7..249c55e 100644
--- a/node_modules/react-native-element-dropdown/lib/module/index.js
+++ b/node_modules/react-native-element-dropdown/lib/module/index.js
@@ -1,7 +1,7 @@
 import Dropdown from './components/Dropdown';
 import MultiSelect from './components/MultiSelect';
 import SelectCountry from './components/SelectCountry';
-import { IDropdownRef } from './components/Dropdown/model';
-import { IMultiSelectRef } from './components/MultiSelect/model';
-export { Dropdown, MultiSelect, SelectCountry, IDropdownRef, IMultiSelectRef };
+// import { IDropdownRef } from './components/Dropdown/model';
+// import { IMultiSelectRef } from './components/MultiSelect/model';
+export { Dropdown, MultiSelect, SelectCountry, /*IDropdownRef, IMultiSelectRef*/ };
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/react-native-element-dropdown/lib/typescript/index.d.ts b/node_modules/react-native-element-dropdown/lib/typescript/index.d.ts
index c7e2a53..96ec0c9 100644
--- a/node_modules/react-native-element-dropdown/lib/typescript/index.d.ts
+++ b/node_modules/react-native-element-dropdown/lib/typescript/index.d.ts
@@ -1,7 +1,7 @@
 import Dropdown from './components/Dropdown';
 import MultiSelect from './components/MultiSelect';
 import SelectCountry from './components/SelectCountry';
-import { IDropdownRef } from './components/Dropdown/model';
-import { IMultiSelectRef } from './components/MultiSelect/model';
+import type { IDropdownRef } from './components/Dropdown/model';
+import type { IMultiSelectRef } from './components/MultiSelect/model';
 export { Dropdown, MultiSelect, SelectCountry, IDropdownRef, IMultiSelectRef };
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/node_modules/react-native-element-dropdown/src/index.tsx b/node_modules/react-native-element-dropdown/src/index.tsx
index bc5f7fc..4267f32 100644
--- a/node_modules/react-native-element-dropdown/src/index.tsx
+++ b/node_modules/react-native-element-dropdown/src/index.tsx
@@ -1,7 +1,7 @@
 import Dropdown from './components/Dropdown';
 import MultiSelect from './components/MultiSelect';
 import SelectCountry from './components/SelectCountry';
-import { IDropdownRef } from './components/Dropdown/model';
-import { IMultiSelectRef } from './components/MultiSelect/model';
+import type { IDropdownRef } from './components/Dropdown/model';
+import type { IMultiSelectRef } from './components/MultiSelect/model';
 
 export { Dropdown, MultiSelect, SelectCountry, IDropdownRef, IMultiSelectRef };
